lib/checkout.js

```diff
@@ -0,0 +1,13 @@
+import { loadStripe } from "@stripe/stripe-js"
+
+const stripe = loadStripe(process.env.NEXT_PUBLIC_STRIPE_PKEY)
+
+export async function initiateCheckout({lineItems} = {}) {
+    const session =  await (await stripe).redirectToCheckout({
+        // payment_method_types: ['card'],
+        lineItems,
+        mode: 'subscription',
+        successUrl: 'http://localhost:3000/success',
+        cancelUrl: 'http://localhost:3000/stripe-checkout',
```

Comment:
Please make this use reference dynamically on the current site URL instead of using `localhost:3000`.

Reply:
@ronix-webriq please take note.